src/core/DisplayGroup.cpp

```diff
@@ -163,6 +176,17 @@ void DisplayGroup::setContentWindows( ContentWindowPtrs contentWindows )
     }
 }
 
+DisplayGroup& DisplayGroup::operator=( const DisplayGroup& displayGroup )
+{
+    setContentWindows( displayGroup.getContentWindows() );
+    setShowWindowTitles( displayGroup.getShowWindowTitles() );
+    coordinates_.setX( displayGroup.getCoordinates().x() );
```

Comment:
Like this, no signals are emitted so you might as well do coordinates_ = displayGroup.getCoordinates(). You need to do setX( displayGroup.getCoordinates().x() ) to emit signals.